confgenerator/logging_receivers.go

```diff
@@ -228,3 +228,28 @@ func (r LoggingReceiverWindowsEventLog) Components(tag string) []fluentbit.Compo
 func init() {
 	LoggingReceiverTypes.RegisterType(func() Component { return &LoggingReceiverWindowsEventLog{} }, "windows")
 }
+
+// A LoggingReceiverSystemd represents the user configuration for a Systemd/journald receiver.
+type LoggingReceiverSystemd struct {
+	ConfigComponent `yaml:",inline"`
+}
+
+func (r LoggingReceiverSystemd) Type() string {
+	return "systemd"
```

Comment:
IMO `systemd` is a bad name for this receiver; systemd supports multiple different logging systems and journald can collect logs from more than just systemd, so I think a bare `systemd` is misleading. I know that's what fluentbit calls it, but I would suggest something like `systemd-journald` or `journald`

Reply:
`systemd_journald` seems like a fine name. Thanks!